Rmd/DESeq2_report.rnaseq.Rmd

```diff
@@ -359,9 +359,18 @@ if(!is.na(params$sortcol)){
 short_contrast_names <- paste(contrasts$V1, "v.", contrasts$V2) # Customize these for your experiment... Must be short enough to fit as Excel tab titles.
 
 # Needed later
-ensembl <- useMart("ensembl",
-                   dataset = ensembl_species,
-                   host = "uswest.ensembl.org")
+ensembl <- NULL
+tryCatch({
+  ensembl <- useMart("ensembl",
+                     dataset = ensembl_species,
+                     host = "uswest.ensembl.org")
+}, error = function(err){
+     ensembl <- useMart("ensembl",
+                     dataset = ensembl_species,
+                     host = "useast.ensembl.org")
+},finally={
+    if (is.null(ensembl)) {stop("Can't connect to Ensembl!")}
+})
```

Comment:
I'm not completely sure I'm doing this right, but it seems to work

I haven't tested a case where both mirrors fail... because I have no control over that lol

Reply:
Doesn't ensembl try both on its own before giving up?